master/buildbot/db/logs.py

```
@@ -157,7 +155,7 @@ def thdGetLogLines(conn) -> str:
             for row in conn.execute(q):
                 # Retrieve associated "reader" and extract the data
                 # Note that row.content is stored as bytes, and our caller expects unicode
-                data = self.COMPRESSION_BYID[row.compressed]["read"](row.content)
+                data = self.COMPRESSION_BYID.get(row.compressed, RawCompressor).read(row.content)
```

We should probably throw an exception if the decompressor is unavailable.